pkg/api/query/v1.go

```diff
@@ -691,6 +706,15 @@ func (qapi *QueryAPI) queryRange(r *http.Request) (interface{}, []error, *api.Ap
 	// Record the query range requested.
 	qapi.queryRangeHist.Observe(end.Sub(start).Seconds())
 
+	queryStr := r.FormValue("query")
+
```

Comment:
Nit but I suggest removing these blank lines before `if` because this is one whole block dealing with `queryStr` hence it looks a bit weird to have this new line here :smile: